package.json

```diff
@@ -50,10 +56,13 @@
     "@nrchkb/logger": "^1.3.3",
     "abortcontroller-polyfill": "^1.7.5",
     "axios": "^1.1.3",
-    "cookie": "0.5.0",
-    "ws": "^8.9.0"
+    "cookie": "^0.5.0",
+    "ws": "^8.9.0",
+    "lodash": "^4.17.21"
```

Comment:
No lodash please :(

Reply:
lodash is used to match one object with another (with varying levels)
this allows matching events from Unifi with a `shapeProfile`

if no lodash - do you have another suggestion?

Reply:
This one is still questionable.

`isMatch` from `lodash` does a really good job at identifying  matching objects no matter how deep the object tree can go